tools/environment-setup.sh

```diff
@@ -14,9 +14,15 @@ if ! grep -q '^name: snapcraft$' snap/snapcraft.yaml; then
 fi
 
 # Create the container
-lxc init ubuntu:16.04 snapcraft-dev
-lxc config set snapcraft-dev raw.idmap "both $UID 1000"
-lxc start snapcraft-dev
+if ! lxc list | grep -q "snapcraft-dev"; then
+    lxc init ubuntu:16.04 snapcraft-dev
+fi
+if ! lxc config get snapcraft-dev raw.idmap | grep -q "both $UID 1000"; then
+    lxc config set snapcraft-dev raw.idmap "both $UID 1000"
+fi
+if ! lxc list | grep "snapcraft-dev" | grep -q "RUNNING"; then
```

Comment:
suggested improvement to improve accuracy:

`if ! lxc info snapcraft-dev | grep -q "Status: Running"; then`